tkg/client/cluster.go

```diff
@@ -180,16 +180,6 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 				return false, nil
 			}
 
-			// if both FeatureFlagPackageBasedLCM and FeatureFlagAutoApplyGeneratedClusterClassBasedConfiguration enabled
```

Comment:
Why is this code piece not required anymore?

Reply:
Because this judgement has been done here: https://github.com/vmware-tanzu/tanzu-framework/blob/2a0a821be32dd7b33c15feb21914d765efa234d0/tkg/client/common.go#L366-L377. During decide if deploy classy or legacy, if ytt customization exists and it's workload cluster, however, allow-legacy-cluster is disabled, then code will return error.